packages/dtslint/dtslint.json

```diff
@@ -14,33 +13,12 @@
 
         "comment-format": [true, "check-space"], // But not check-uppercase or check-lowercase
         "interface-name": [true, "never-prefix"],
-        "max-line-length": [true, 200],
```

Comment:
Kinda funky that we have 200 here but 120 in the prettier config (and now dprint config). Of course, I personally use 120 so 🤫